test/CMakeLists.txt

```diff
@@ -1,22 +1,20 @@
 itk_module_test()
-set(SkullStripTests
-SkullStrippingTest.cxx
-)
 
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../examples)
+set(SkullStripTests
+  itkStripTsImageFilterTest.cxx
+  )
 
-CreateTestDriver(SkullStrip  "${SkullStrip-Test_LIBRARIES}" "${SkullStripTests}")
+CreateTestDriver(SkullStrip "${SkullStrip-Test_LIBRARIES}" "${SkullStripTests}")
 
-set(TEMP ${ITK_TEST_OUTPUT_DIR})
-set(TEST_DATA_ROOT "${SkullStrip_SOURCE_DIR}/test")
-itk_add_test( NAME SkullStrippingTest
+itk_add_test(NAME itkStripTsImageFilterTest
   COMMAND SkullStripTestDriver
-    --compare ${TEST_DATA_ROOT}/Baseline/outputMaskBaseline.mha
-              ${TEMP}/outputMask.mha
-    SkullStrippingTest
-    ${TEST_DATA_ROOT}/Input/brainweb.mha
-    ${TEST_DATA_ROOT}/Input/atlasImage.mha
-    ${TEST_DATA_ROOT}/Input/atlasMask.mha
-    ${TEMP}/outputMask.mha
-    ${TEMP}/outputMaskedImage.mha
-)
+  --compare-MD5
+    ${ITK_TEST_OUTPUT_DIR}/outputMask.mha
+    "dad6e6dd8b849b884d15e91954f808d1"
```

Comment:
The MD5 link is not the one corresponding to this test: I wait for the CI's to tell me the good one so that I can fix it.

Reply:
It also makes later debugging harder. And including the baseline image makes less sense if has comparison is used instead of image comparison.

Reply:
I do share that different viewpoints exist on this. We had a similar discussion a [while back ago](http://review.source.kitware.com/#/c/23550/), and we did not reach consensus. I am open to changing it and adopting any new practice if a decision is made.

Reply:
Without consensus, the person doing the work get to decide what to do 😄

Reply:
+1 to leaning towards image comparisons, especially without a deterministic, integer-based computation.

Reply:
Granted. 65b622b uses direct image comparison. However, wrapping issues preclude CI's from being green. PR #10 was opened to fix the wrapping issues.